SharedTests/NSURLExtensionsTests.swift

```diff
@@ -9,25 +9,61 @@ import Shared
 class NSURLExtensionsTests : XCTestCase {
     func testRemovesHTTPFromURL() {
         let url = NSURL(string: "http://google.com")
-        if let actual = url?.absoluteStringWithoutHTTPScheme() {
+        if let actual = url?.absoluteDisplayString() {
             XCTAssertEqual(actual, "google.com")
         } else {
             XCTFail("Actual url is nil")
         }
     }
 
+    func testRemovesHTTPAndTrailingSlashFromURL() {
+        let url = NSURL(string: "http://google.com/")
+        if let actual = url?.absoluteDisplayString() {
+            XCTAssertEqual(actual, "google.com")
+        } else {
+            XCTFail("Actual url is nil")
+        }
+    }
+
+    func testRemovesHTTPButNotTrailingSlashFromURL() {
+        let url = NSURL(string: "http://google.com/foo/")
+        if let actual = url?.absoluteDisplayString() {
+            XCTAssertEqual(actual, "google.com/foo/")
+        } else {
+            XCTFail("Actual url is nil")
+        }
+    }
+
     func testKeepsHTTPSInURL() {
         let url = NSURL(string: "https://google.com")
-        if let actual = url?.absoluteStringWithoutHTTPScheme() {
+        if let actual = url?.absoluteDisplayString() {
             XCTAssertEqual(actual, "https://google.com")
         } else {
             XCTFail("Actual url is nil")
         }
     }
 
+    func testKeepsHTTPSAndRemovesTrailingSlashInURL() {
+        let url = NSURL(string: "https://google.com/")
+        if let actual = url?.absoluteDisplayString() {
+            XCTAssertEqual(actual, "https://google.com")
+        } else {
+            XCTFail("Actual url is nil")
+        }
```

Comment:
We generally just use force unwrapping in tests, so you could do `XCTAssertEqual(url!.absoluteDisplayString(), "https://google.com")` without any of the `if`/`else` stuff (and it will throw/fail either way). But I guess this is fine too if you prefer it.